src/backend/api/Fusion.Resources.Api/Controllers/Departments/DepartmentsController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Fusion.Resources.Domain;
+using Fusion.Resources.Domain.Queries;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Api.Controllers.Departments
+{
+    [ApiVersion("1.0-preview")]
+    [Authorize]
+    [ApiController]
+    public class DepartmentsController : ResourceControllerBase
+    {
+        [HttpGet("resources/departments/{departmentString}")]
```

Comment:
I don't think we have any other uris formatted like this in the api? The other department related endpoints just use `/departments` on the root path.